lib/auth/join_github.go

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
This looks off to me. Is the whitespace different compared to the license in other files?

Reply:
Hm yeah looks like my clipboard clobbered the newlines out.

Reply:
0bf20c749